src/android/patches/makeStringsPatch.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function makeStringsPatch(params) {
+  const patch = Object.keys(params).map(key =>
+    `    <string moduleConfig="true" name="${key}">${params[key]}</string>`
```

Comment:
> Yeah, that's why I used it, actually :) I think if the approach with splitting vars by type wouldn't work, I'd say we need to go with rnpm.xml approach, but I want to give it a chance first.

I still believe it might be easier to go with a separate file, otherwise it will be tricky to update (you will have to either support backwards `strings.xml`, write a migration or bump a major). That sounds like a 5 minutes tweak that will be more future proof.